content/en/blog/2024/otel-get-started-survey/index.md

```diff
@@ -0,0 +1,77 @@
+
+Check out the results!
+
+
+## Key Takeaways
+* No matter their level of OTel experience, folks would like to see the following:
+    * #1 Comprehensive documentation (67.3% of respondents)
+    * #2 Reference implementations for instrumentation (65.3% of respondents)
+    * #3 Detailed tutorials (63.4% of respondents)
+* Respondents are mostly working with containers, with 79.8% of respondents using Kubernetes and 63.5% of respondents using Docker
+* Over 83% of respondents stated that the Traces Specification, Instrumentation APIs & SDKs, and Metrics Specification were the most important in their OTel journey.
+
+
+## Detailed insights
+### Q: What technologies are you using when setting up your observability journey? 
+
+Out of 104 respondents: Respondents are mostly working with containers, with 79.8% of respondents using Kubernetes and 63.5% of respondents using Docker
+
+![Chart showing what technologies are being used](1-technologies-used.png)
+
+### Q: Where in your Observability journey is your organization?
+
+Out of 104 respondents:
+* 25% were beginners and they are learning about observability or have used monitoring tools
+* 49% were intermediates and were setting up observability practice 
+* 26% were experts and have a well-established observability practice
+
+![Chart showing where in their observability journey they are](2-level.png)
+
+### Q: What languages are you using in your organization?
+
+JavaScript, Java, Go, and Python are the most widely used languages, they are being utilized by over 50% of the respondents.
+
+Other languages used included Scala, TypeScript, Erlang/Elixir, Perl, R, Lua, Matlab, Julia, Haskell, COBOL, Kotlin, and SAS.
+
+![Chart showing what languages are being used](3-languages.png)
+
+### Q: What resources do you wish you had when getting started with OpenTelemetry?
+
+Depending on where in their observability journey they prefer different resources: 
+
+
+* Those who are learning about observability and those who are setting up an observability practice want comprehensive documentation and detailed tutorials and videos. 
+* Those who have a well-established observability practice prefer reference implementations for instrumentation as well as comprehensive documentation. 
+
+![Chart showing what resources folks want](4-resources.png)
+
+
+### Q: What part of the project is most important to you in your OTel journey?
+
+Most respondents stated that Traces Specification, Instrumentation APIs & SDKs and Metrics Specification were considered the most important.
+
+
+![Chart showing what aspects are the most important](5-importance.png)
```

Comment:
Quite interesting to see that only few people consider LLMs as important to their OTEL journey.
Wondering if it would be nice to point that out too..

Reply:
I also find really interesting that specification and APIs are above the Collector. Sometimes, for those working in observability engineering circles, it feels like the Collector is currently what end-users care the most about (and I'm not saying with this it's not crucial for their operations). Maybe there's something to add here about this fact , which illustrates the importance of OpenTelemetry as a cross-cutting concern within application code?

I think the rest looks great, so I'll leave my approval anyway. This is just a suggestion and feel free to ignore.

Reply:
> Quite interesting to see that only few people consider LLMs as important to their OTEL journey. Wondering if it would be nice to point that out too..

Re: LLMs, I would say that although LLMs were low on the OTel journey scale, this may change in the coming months and years, as LLMs continue to gain traction.

Reply:
> I also find really interesting that specification and APIs are above the Collector. Sometimes, for those working in observability engineering circles, it feels like the Collector is currently what end-users care the most about (and I'm not saying with this it's not crucial for their operations). Maybe there's something to add here about this fact , which illustrates the importance of OpenTelemetry as a cross-cutting concern within application code?
> 
> I think the rest looks great, so I'll leave my approval anyway. This is just a suggestion and feel free to ignore.

Definitely an interesting observation; however, we should probably stick to the facts here, and not try to insert opinion in this type of piece. Just my two cents. 😄

Reply:
Thinking the same around LLMs, would rather leave as is, and with the collector  also leave as is. Maybe we just make sure to post the results on each of the corresponding SIG's (for the importance question) slack channels to give them the heads up that this information is available

Reply:
I'm fine with that. @AnaMMedina21 can you raise an issue to capture this, and also post the results to the various SIGs?

Reply:
@avillela listo https://github.com/open-telemetry/sig-end-user/issues/43!